doc/source/cookbook/circle.reg

```diff
@@ -0,0 +1,4 @@
+# Region file format: DS9 version 4.1
```

Comment:
Should this file be included here? Looks like a file that might get generated when the docs build?

Reply:
Oh yeah, I think it is actually. Ugh. I'll remove it.

Reply:
removed